all_test.go

```diff
@@ -259,25 +271,31 @@ func TestResetPassword(t *testing.T) {
 	}
 
 	// Test time expiry
-	token3, _ := c.ResetPasswordStart("joe", time.Now().Add(-3*time.Second))
-	if err := c.ResetPasswordFinish("joe", token3, "12345"); err != ErrPasswordTokenExpired {
+	token3, _ := c.ResetPasswordStart(joeUserId, time.Now().Add(-3*time.Second))
+	if err := c.ResetPasswordFinish(joeUserId, token3, "12345"); err != ErrPasswordTokenExpired {
 		t.Fatalf("ResetPasswordFinish should have failed with ErrPasswordTokenExpired instead of %v", err)
 	}
+
+	teardown(c)
 }
 
 func TestBasicAuth(t *testing.T) {
 	c := setup1(t)
 
 	expect_username_password := func(username, password, expectErrorStart, expectIdentity string) {
+		//authenticate
```

Comment:
We always leave a single space after `//`. I did mention this in the previous round of comments. The previous issue was also mentioned in the previous round of comments.